content/nim/monitoring/security-monitoring/give-access-to-security-monitoring-dashboards.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Add user access to Security Monitoring dashboards
+toc: true
+weight: 200
+doctype: how-to
```

Comment:
```suggestion
type: how-to
```

Reply:
Yes, which I added to the archetypes in the Hugo theme two months ago.

https://github.com/nginxinc/nginx-hugo-theme/tree/main/archetypes

My working PR for migrating NGF has this same format for almost every single page, and I have also been proactively making these changes to every single page I've worked on since.

Reply:
> Yes, which I added to the archetypes in the Hugo theme two months ago.

How would we know to look there? Ideally, we'd automate the process with some sort of linter.